mediapipe/graphs/instantmotiontracking/calculators/gl_animation_overlay_calculator.cc

```diff
@@ -349,8 +351,6 @@ bool GlAnimationOverlayCalculator::LoadAnimationAndroid(
 
     // Used for storing the vertex normals prior to averaging
     float vertex_normals_sum[lengths[0]];
-    int vertex_denom[lengths[0]];
-
     // Compute every triangle surface normal and store them for averaging
```

Comment:
I didn't realize we were copying all of our vertex normal calculation code.  We should pull this all to its own method in that case (to avoid duplication between lines 354-411 and 483-540; that's a lot of code to maintain in two places).